.github/workflows/tests.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       fail-fast: false
       matrix:
         # https://ruby-lang.org/en/downloads/branches
-        ruby: ["3.3", "3.2", "3.1"]
+        ruby: ["3.3", "3.2"]
```

Comment:
I think we should add 3.4 due to the phrasing in the README.md file for other versions - some examples:
> Ruby 3.1.0+
> Ruby 3.0.0+
> Ruby 2.7.0+